data/geany.css

```diff
@@ -36,6 +36,17 @@
 	color: #007f00;
 }
 
+/* compiler message colors */
+#geany-compiler-error {
+    color: #ff0000;
+}
+#geany-compiler-context {
+    color: #880000;
```

Comment:
shouldn't that be `0x7f0000` to be the same as the hardcoded one?  Not that it matters too much though, I don't imagine anybody will really see the difference